### DISCUSSION TITLE
angg.twu.net or anggtwu.net ?

### DISCUSSION THREAD

**Discussion by martin12333** [edrx/eev#6] [Q&A] [answered]
should i link to

http://angg.twu.net     , or <br/>
http://anggtwu.net

?

EDIT: probably *anggtwu* 
https://github.com/edrx/eev/commits/UTF-8

**Comment by edrx** [comment, marked as answer]
Anggtwu is better! See:

http://anggtwu.net/angg.html

**Reply by martin12333** [reply]
@edrx btw, about web hosting:

there are several very low cost options for web hosting, e.g.

https://sdf.org/?join

 ... a bit of nostalgia for me // I started reading a book about Unix in 1991

https://en.wikipedia.org/wiki/SDF_Public_Access_Unix_System
